modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/database/H2TreeIndexBase.java

```diff
@@ -38,8 +41,15 @@
  * H2 tree index base.
  */
 public abstract class H2TreeIndexBase extends GridH2IndexBase {
+
     /** Default value for {@code IGNITE_MAX_INDEX_PAYLOAD_SIZE} */
-    static final int IGNITE_MAX_INDEX_PAYLOAD_SIZE_DEFAULT = 10;
+    static final int IGNITE_MAX_INDEX_PAYLOAD_SIZE_DEFAULT = 64;
+
+    /** Default sql index size for types with variable length (such as String or byte[]). */
+    static final int IGNITE_VARIABLE_TYPE_DEFAULT_INDEX_SIZE = 10;
```

Comment:
By the way, is there a way to tune this? IMO, it worth to provide a way to adjust this param somehow.

Reply:
@ygerzhedovich Do you agree with such update?

Reply:
yes, however it should be provide through decision approval procedures - I think it could be done in separate ticket in case it will require significant changes